circle.yml

```diff
@@ -0,0 +1,3 @@
+machine:
+    python:
+        version: 2.7.3
```

Comment:
Probably a blind attempt at getting cci to work until I realized the keys were bad. Will revert